src/SearchEndpoint/HighlightEndpoint.php

```diff
@@ -37,13 +37,13 @@ class HighlightEndpoint extends AbstractSearchEndpoint
     /**
      * {@inheritdoc}
      */
-    public function normalize(NormalizerInterface $normalizer, string $format = null, array $context = [])
+    public function normalize(NormalizerInterface $normalizer, string $format = null, array $context = []): array|string|int|float|bool
     {
         if ($this->highlight) {
             return $this->highlight->toArray();
         }
 
-        return null;
```

Comment:
Why not null?

Reply:
Cause we have to match the interface which defines non-nullable response